docs/site-components-deployment.md

```diff
@@ -0,0 +1,548 @@
+-rw-r--r-- 1 tf2pickup tf2pickup  29100584 Jan 14  2019 cp_logjam_rc9.bsp
+-rw-r--r-- 1 tf2pickup tf2pickup  12987328 Jan 14  2019 cp_logjam_rc9.bsp.bz2
+-rw-r--r-- 1 tf2pickup tf2pickup  18970621 Dec  8  2019 cp_metalworks.bsp
+-rw-r--r-- 1 tf2pickup tf2pickup  18687571 Dec  8  2019 cp_metalworks.bsp.bz2
+-rw-r--r-- 1 tf2pickup tf2pickup  18312502 Nov 29  2020 cp_process_f7.bsp
+-rw-r--r-- 1 tf2pickup tf2pickup  18024376 Nov 29  2020 cp_process_f7.bsp.bz2
+-rw-r--r-- 1 tf2pickup tf2pickup  49282532 Apr 15  2017 cp_process_final.bsp
+-rw-r--r-- 1 tf2pickup tf2pickup  21364357 Apr 15  2017 cp_process_final.bsp.bz2
+-rw-r--r-- 1 tf2pickup tf2pickup  77921396 Jul 22  2018 cp_prolands_b6.bsp
+-rw-r--r-- 1 tf2pickup tf2pickup  23282904 Jul 22  2018 cp_prolands_b6.bsp.bz2
+-rw-r--r-- 1 tf2pickup tf2pickup  72041100 Jan  3  2019 cp_prolands_rc2p.bsp
+-rw-r--r-- 1 tf2pickup tf2pickup  31122834 Jan  3  2019 cp_prolands_rc2p.bsp.bz2
+-rw-r--r-- 1 tf2pickup tf2pickup  32477488 Apr 25  2020 cp_reckoner_rc6.bsp
+-rw-r--r-- 1 tf2pickup tf2pickup  32243941 Apr 25  2020 cp_reckoner_rc6.bsp.bz2
+-rw-r--r-- 1 tf2pickup tf2pickup  20378888 Jan 31  2019 cp_snakewater_final1.bsp
+-rw-r--r-- 1 tf2pickup tf2pickup  20126408 Jan 31  2019 cp_snakewater_final1.bsp.bz2
+-rw-r--r-- 1 tf2pickup tf2pickup  30319857 Dec  8  2019 cp_sunshine.bsp
+-rw-r--r-- 1 tf2pickup tf2pickup  29938052 Dec  8  2019 cp_sunshine.bsp.bz2
+-rw-r--r-- 1 tf2pickup tf2pickup  19277592 Feb  7 17:44 koth_clearcut_b15d.bsp
+-rw-r--r-- 1 tf2pickup tf2pickup   9010068 Feb  7 17:44 koth_clearcut_b15d.bsp.bz2
+-rw-r--r-- 1 tf2pickup tf2pickup  17218901 May 11  2019 koth_product_rcx.bsp
+-rw-r--r-- 1 tf2pickup tf2pickup  17056131 May 11  2019 koth_product_rcx.bsp.bz2
+```
+
+Then, these are templates for the aforementioned files:
+
+## `.env`
+
+```env
+### Configuration for the Server
+
+# An URL to where this server instance will be accessed
+API_URL=https://api.tf2pickup.fi
+
+# An URL to where the client is hosted
+CLIENT_URL=https://tf2pickup.fi
+
+# The bot name; it can be changed via the web client afterwards
+BOT_NAME=tf2pickup.fi
+
+# MongoDB
+MONGODB_HOST=localhost
+MONGODB_PORT=8001
+MONGODB_DB=admin
+MONGODB_USERNAME=tf2pickup
+MONGODB_PASSWORD=yoursuperfunnypassword
+
+# Steam API key
+# Get your key at https://steamcommunity.com/dev/apikey
+STEAM_API_KEY=1234567890ABCDEF1234567890ABCDEF
+
+# A passphare that is used to encrypt the keystore file.
+# NOTE: For production, get a random password (i.e. from https://passwordsgenerator.net/)
+# and do not change it afterwards. If you change the passphare all JWT tokens will no longer
+# be valid, so all clients will be logged out.
+KEY_STORE_PASSPHARE=XDXDXDXDXDXDXDXDXD
+
+# SteamID of the super-user
+# NOTE: Use the SteamID64 format.
+SUPER_USER=76561198011558250
+
+# Which gamemode to run; see src/configs/queue for different gamemodes, possible values: 6v6, 9v9
+QUEUE_CONFIG=6v6
+
+# Mumble channel configuration
```

Comment:
Mumble channel configuration is no longer done via the _.env_ file. This part of the documentation will be moved to the website itself.